api/lib/routes/scalingHistories.js

```diff
@@ -0,0 +1,44 @@
+"use strict"
+module.exports = function(settings) {
+  var express = require("express");
+  var router = express.Router();
+  var logger = require("../log/logger");
+  var HttpStatus = require("http-status-codes");
+  var scalingEngineUtil = require("../utils/scalingEngineUtils")(settings.scalingEngine);
+  var scalingHistoryHelper = require("./scalingHistoryHelper");
+
+  router.get("/:guid/scaling_histories", function(req, resp) {
```

Comment:
can we use app_guid `/:app_guid/scaling_histories` 
just to make clear difference from policy_guid